packages/core/src/components/private/FormInput.tsx

```diff
@@ -26,12 +29,14 @@ export type IgnoreAttributes =
   | 'security';
 
 export type Props<T = any> = {
-  /** Decrease font size and padding. */
+  /** Decrease font size and padding to small. */
   compact?: boolean;
   /** Mark the field as important. */
   important?: boolean;
   /** Mark the field as invalid. */
   invalid?: boolean;
+  /** Increase font size and padding to large. */
+  large?: boolean;
```

Comment:
We originally had small/large for all forms, but we decided we never wanted a large size, hence why forms use `compact` and not `small`. What's the use case for needing a large?